projects/igniteui-angular/src/lib/select/select.component.ts

```diff
@@ -118,7 +118,7 @@ export class IgxSelectComponent extends IgxDropDownComponent implements IgxSelec
      * An @Input property that sets input placeholder.
      *
      */
-    @Input() public placeholder = '';
+    @Input() public placeholder;
```

Comment:
Why don't you just host bind the attribute placeholder?

Reply:
For the placeholder to work as expected, I need to pass it to the actual input element. So I am using an igxSelect   **@input** named as well "**placeholder**" to pass the user defined one to the **input's** placeholder attribute. 

Using the  **attr.placeholder** is needed, because otherwise the **@input** is evaluated as string and thus from the component's perspective there is always a placeholder (and the corresponding styles applied) even if it is an **empty string**, **null** or **undefined**.